From 60b9ade309e8a7534117eb0e5a2736a59ac7d463 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sun, 4 Feb 2018 21:53:22 +0200
Subject: [PATCH 9/9] SUW: Fix keydisabler setting on non-gms case

* Fixes:
  W LineageSettings: Setting dev_force_show_navbar has moved from
  LineageSettings.Secure to LineageSettings.Global, value is unchanged.

Change-Id: Id22886ea5767c6adf254618ecec84b338e11979b
---
 src/org/lineageos/setupwizard/LineageSettingsActivity.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/org/lineageos/setupwizard/LineageSettingsActivity.java b/src/org/lineageos/setupwizard/LineageSettingsActivity.java
index d5e2bfb..5970930 100644
--- a/src/org/lineageos/setupwizard/LineageSettingsActivity.java
+++ b/src/org/lineageos/setupwizard/LineageSettingsActivity.java
@@ -209,8 +209,8 @@ public class LineageSettingsActivity extends BaseSetupWizardActivity {
     private void updateDisableNavkeysOption() {
         if (!mHideNavKeysRow) {
             final Bundle myPageBundle = mSetupWizardApp.getSettingsBundle();
-            boolean enabled = LineageSettings.Secure.getInt(getContentResolver(),
-                    LineageSettings.Secure.DEV_FORCE_SHOW_NAVBAR, 0) != 0;
+            boolean enabled = LineageSettings.Global.getInt(getContentResolver(),
+                    LineageSettings.Global.DEV_FORCE_SHOW_NAVBAR, 0) != 0;
             boolean checked = myPageBundle.containsKey(DISABLE_NAV_KEYS) ?
                     myPageBundle.getBoolean(DISABLE_NAV_KEYS) :
                     enabled;
-- 
2.7.4

